usersession/client/client_test.go

```diff
@@ -126,7 +126,7 @@ func (s *clientSuite) TestAgentTimeout(c *C) {
 
 	// An error is reported, and we receive information about the
 	// agent that replied on time.
-	c.Assert(err, ErrorMatches, `Get http://1000/v1/session-info: context deadline exceeded`)
+	c.Assert(err, ErrorMatches, `Get \"http://1000/v1/session-info\": context deadline exceeded`)
```

Comment:
Should this have `"?` as the rest above does?

Reply:
Yes, I'm a muppet. Fixed and forced pushed (so that I can easier cherry pick into 2.44). Thanks for spotting!